lib/iamvery/phoenix/live_view/test_helpers.ex

```diff
@@ -63,17 +64,13 @@ defmodule Iamvery.Phoenix.LiveView.TestHelpers do
         {conn, {:ok, view, html}}
       end
 
-      def submit_form({conn, {:ok, view, _html}}, selector, attributes, opts \\ []) do
+      def submit_form({conn, {:ok, view, _html}}, selector, attributes) do
         html =
           view
           |> form(selector, attributes)
           |> render_submit()
 
-        if Keyword.get(opts, :follow, true) do
-          {conn, follow_redirect(html, conn)}
-        else
-          {conn, {:ok, view, html}}
-        end
+        {conn, follow_redirect(html, conn)}
```

Comment:
🤔 There may still be a use case for submitting forms w/o redirecting, but I haven't seen one in a production setting yet, so I'm going to punt until there's a requirement.